mac

```diff
@@ -46,14 +46,16 @@ fi
 # shellcheck disable=SC2016
 append_to_zshrc 'export PATH="$HOME/.bin:$PATH"'
 
-if [ -d "/usr/local" ]; then
-  if ! [ -r "/usr/local" ]; then
-    sudo chown -R "$(whoami)":admin /usr/local
+HOMEBREW_PREFIX="/usr/local"
+
+if [ -d "$HOMEBREW_PREFIX" ]; then
+  if ! [ -r "$HOMEBREW_PREFIX" ]; then
+    sudo chown -R "$USER:admin" /usr/local
```

Comment:
You'll want `$LOGNAME` instead of `$USER` - some details on this commit message: https://github.com/thoughtbot/rcm/commit/a801c4c58902cae7b835d890487edc39bd7da142